data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,946 @@
+			`	You give it a moment of thought.`
+			choice
+				`	"That seems practical."`
+			`	"Fantastic," she says, though her tone is much more business like than the wording would imply. "In that case, if you could please take this to Earth and communicate with your various governments we can begin a dialogue on how best to deal with this current scenario."`
+			choice
+				`	"Various? We only have two governments."`
+			`	"Oh," she says, "we've treated the Syndicate as a de facto government for many decades now; with the amount of influence they have it was the only prudent thing to do."`
+			`	You take the boxed item with you, and an escort awaits to return you to your ship. As the door behind you closes, you hear Sayari mutter to someone, "It really was much easier when they just had one government though. What a mess."`
+				accept
+	
+	
+mission "Hai Leaks Response 2"
+	landing
+	name "Collect Representatives"
+	description "Collect human government representatives from <stopovers> and take them to <destination>."
+	source "Earth"
+	stopover "Deep"
+	stopover "Foundry"
+	destination "Hai-home"
+	blocked "You have a mission here for the Hai, but you need three bunks free."
+	clearance
+	to offer
+		has "Hai Leaks Response 1: done"
+	passengers 3
+	on offer
+		event "pirates respond to leaks"
+		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
+		conversation
+			`On approach to the spaceport on Earth, you request to speak to an official of significant standing on diplomatic business. Thanks to your renown from the Pug war, you are able to shortcut objections stemming from you yourself not being a diplomat. The diplomatic ID from the box Sayari gave you doesn't hurt either.`
+			`	The Republic Deputy Minister for Foreign Affairs meets you in a private room at the spaceport.`
+			`	"Captain <last>," he steps around the table to grasp your hand in a firm handshake. "Xilin Yang. It's an honor to meet a war hero like yourself."`
+			choice
+				`	"It was hard-fought, but it was necessary."`
+				`	"Thanks, but it wasn't as dramatic as they made it out to be. I'm just happy to have helped where I could."`
+					goto humble
+			`	You play the part and try to act noble. Mr. Yang seems more than happy to embody respect for the hero of his preferred narrative.`
+			`	"You're every inch what I expected," he assures you before moving to take his seat.`
+				goto sit
+			label humble
+			`	You shrug, palms up, and explain, "You should listen to the xenoanthropologists. The Pug were just here to test us, to make us really consider what mattered and come together."`
```

Comment:
This felt like a bit of a non sequitur, and I was confused for a while until I realised he was talking about the war with the Pug rather than the war with the Free Worlds. Perhaps it would make more sense if he described you as "the hero who defeated the Pug" rather than "a war hero"?

Reply:
Given that most of the fighting was against the Pug and that there were only a small handful of conflicts between the FW and Navy, that kinda seemed obvious to me. I'll see about making this more clear though.

Reply:
For me, the length of time spent in the "FW vs Republic" part of the war outweighs the relatively brief "everyone vs Pug" segment, regardless of the amount of actual combat involved. Either way, the player could have put the game down for a month between sessions and forgotten a lot of stuff -- I do this myself rather more often than I like.